conf/application-json-logger.xml

```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<configuration>
-
-    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder class="uk.gov.hmrc.play.logging.JsonEncoder"/>
-    </appender>
-
-    <logger name="uk.gov" level="WARN"/>
-
-    <logger name="api-notification-pull" level="INFO"/>
```

Comment:
The default is [Warn] you need to update the config for [Info] for the API-notification-pull